packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.tsx

```diff
@@ -459,6 +459,7 @@ export class ContextualMenu extends BaseComponent<IContextualMenuProps, IContext
 
     if (ev.which === openKey) {
       this._onItemSubMenuExpand(item, ev.currentTarget as HTMLElement);
+      ev.preventDefault();
```

Comment:
I don't think we should blindly use both of them. `stopPropagation` can be hazardous as evident from the example by @trevorsg. Infact, we just fixed a bug in Dropdown to avoid stopPropgation if it doesn't handle the keypress and give chance to parent components to act on it. 
`preventDefault` is mostly used when we want to avoid browser default behavior and handle it ourselves. As in this example, with custom context menu we don't want browsers to open its default context menu for some element. Or say, we want to avoid link click navigation and handle it ourselves. That said, `preventDefault` and `defaultPrevented` can be used as an alternative to `stopPropagation`. It explained at [dangers of stopping event propagation ](https://css-tricks.com/dangers-stopping-event-propagation/) as one of the best practices to follow.